eq-author/src/components/CommentsPanel/index.js

```diff
@@ -0,0 +1,202 @@
+import React, { useState } from "react";
+import { withRouter } from "react-router";
+import { get } from "lodash";
+import moment from "moment";
+import { useQuery, useMutation } from "@apollo/react-hooks";
+import Loading from "components/Loading";
+import Error from "components/Error";
+import CustomPropTypes from "custom-prop-types";
+
+import COMMENT_QUERY from "./commentsQuery.graphql";
+import COMMENT_ADD from "./createNewComment.graphql";
+
+import styled from "styled-components";
+import ScrollPane from "components/ScrollPane";
+import { colors } from "constants/theme";
+
+import TextArea from "components/Forms/TextArea";
+import { Field, Label } from "components/Forms";
+import Button from "components/buttons/Button";
+import ButtonGroup from "components/buttons/ButtonGroup";
+
+const CommentsPane = styled.div`
+  background: ${colors.white};
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  padding: 0;
+  margin: 0;
+  font-size: 1em;
+`;
+
+const CommentSection = styled.div`
+  padding: 0.7em 1em;
+  border-bottom: 1px solid ${colors.lightMediumGrey};
+`;
+
+const ShowComment = styled.div`
+  align-items: left;
+  padding: 0.5em 1em;
+  margin-bottom: 5px;
+  border: 1px solid ${colors.lightGrey};
+  background-color: ${colors.lighterGrey};
+`;
+
+const CommentHeaderContainer = styled(Field)`
+  display: flex;
+  justify-content: flex-start;
+  -webkit-align-items: start;
```

Comment:
Is there a reason why you use the newer version of `align-items` within `ShowComment` but the older version within `CommentHeaderContainer`? Should the old and new version be included in each, respectively, as well as what already exists?